doc/cabal-package-description-file.rst

```diff
@@ -318,20 +318,33 @@ describe the package as a whole:
     tools require the package-name specified for this field to match
     the package description's file-name :file:`{package-name}.cabal`.
 
-    Package names are case-sensitive and must match the regular expression
-    (i.e. alphanumeric "words" separated by dashes; each alphanumeric
-    word must contain at least one letter):
-    ``[[:digit:]]*[[:alpha:]][[:alnum:]]*(-[[:digit:]]*[[:alpha:]][[:alnum:]]*)*``.
+    A valid package name comprises an alphanumeric 'word'; or two or more
+    such words separated by a hyphen/minus character (``-``). A word cannot be
```

Comment:
```suggestion
    such words separated by a hyphen (``-``). A word cannot be
```

Reply:
I'll put that through. My original text was influenced by the fact that Unicode name for `U+2D` is `HYPHEN-MINUS` and I was aiming for accuracy.